src/model_repository_manager/model_repository_manager.cc

```diff
@@ -1401,17 +1434,21 @@ ModelRepositoryManager::InitializeModelInfo(
     // the override while the local files may still be unchanged.
     linfo->mtime_nsec_ = std::make_pair(0, 0);
     linfo->model_path_ = location;
+    linfo->model_config_path_ = JoinPath({location, kModelConfigPbTxt});
     linfo->agent_model_list_.reset(new TritonRepoAgentModelList());
     linfo->agent_model_list_->AddAgentModel(std::move(localize_agent_model));
   } else {
+    linfo->model_config_path_ = GetModelConfigFullPath(linfo->model_path_);
```

Comment:
Will this get updated periodically? Say if a `model_config_path_` is set and the file get deleted by the user. Will the usage of `model_config_path_` just always pointing to invalid path instead of falling back to some config based on fall back policy?

Reply:
If `model-management-mode` is set to none, the configuration only loaded once per my understanding.

The `InitializeModelInfo` function is invoked by function `Poll`, which is invoked by `LoadUnloadModels`. So if If `model-management-mode` is set to explicit or poll, the configuration location will update. See tests [qa/L0_lifecycle/test.sh](https://github.com/triton-inference-server/server/pull/7185/files#diff-8ad9d754d4d301c75f1892863f7a7ad02287a882d18bff7460b14faac9cc06a6) and [qa/L0_lifecycle/lifecycle_test.py](https://github.com/triton-inference-server/server/pull/7185/files#diff-27572619e19393af9b72b568ffb9d6b2c4932944aa3141a492176636acb8e508).